### PR TITLE
cleanup: add third_party/js.bzl format to ignore

### PR DESCRIPTION
**Pull request by stephanwlee**
Tested by configuring git to use the .git-blame-ignore-revs and `git blame third_party/js.bzl`.

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -83,4 +83,7 @@ ae32a4adcab8c3349c3bff21240078334fef1371  # black: reformat directory tensorboar
 1ccf3fe1c122cd89cde577ffc53b31b9b4e5ee20  # black: reformat directory tensorboard/examples/plugins/example_basic/tensorboard_plugin_example
 
 # Small BUILD reformat (<https://github.com/tensorflow/tensorboard/pull/3054>)
-ea4e9bbc885784a283b8b79974132df7c6cdcc50  # Reformat all `*BUILD` files with `buildifier`
+ea4e9bbc885784a283b8b79974132df7c6cdcc50  # buildifier: Reformat all `*BUILD` files
```
Comment (wchargin) [inline .git-blame-ignore-revs]: nit: The other lines in this file have the full commit hash followed by
the commit headline; could you revert the changes to this line and make
the newly added line follow the same form?

You can do this automatically with:

```
$ git log -1 --format='%H  # %s' 3e17c44d5bc3
3e17c44d5bc3c7af72f14059e39d4b872e95d573  # Buildifier format js.bzl for future edits (#3332)
```

Comment (stephanwlee) [reply .git-blame-ignore-revs]: I didn't know that was the pattern; fixed and reverted  the change on ea4e9bbc885784a283b8b79974132df7c6cdcc50 line.

Comment (wchargin) [reply .git-blame-ignore-revs]: Yeah—it’s generally advised to accompany Git hashes by either the commit
date or the headline to make it easier to identify the object. This
matters more with abbreviated hashes, because more objects in the repo
might actually make the short hash ambiguous. With full hashes, it’s
just courtesy, but it’s still useful: for instance, a few days ago I ran

```
git grep event_processing origin/master:.git-blame-ignore-revs
```

to quickly look up a commit, and that did the right thing. :-)

File: tensorboard/components/vz_chart_helpers/plottable-interactions.js [hunk 2]
```diff
@@ -200,6 +200,15 @@ var vz_chart_helpers;
...
 
+  Plottable.Interaction.prototype._isInsideComponent = function(p) {
+    return (
+      0 <= p.x &&
+      0 <= p.y &&
+      p.x < this._componentAttachedTo.width() &&
+      p.y < this._componentAttachedTo.height()
+    );
+  };
+
```
Comment (wchargin) [inline tensorboard/components/vz_chart_helpers/plottable-interactions.js]: revert?

Comment (stephanwlee) [reply tensorboard/components/vz_chart_helpers/plottable-interactions.js]: Yup, thanks. I forgot which branch I was in :\
